assets/admin/editor-wizard/steps/course-details-step.js

```diff
@@ -21,30 +20,31 @@ import senseiProUpsell from '../../../images/sensei-pro-upsell.png';
  * @param {Function} props.setData
  */
 const CourseDetailsStep = ( { data: wizardData, setData: setWizardData } ) => {
-	usePostTitle( wizardData.courseTitle );
+	const { editPost } = useDispatch( editorStore );
 
-	const updateCourseTitle = ( value ) => {
-		setWizardData( { ...wizardData, courseTitle: value } );
+	const updateCourseTitle = ( title ) => {
+		setWizardData( { ...wizardData, courseTitle: title } );
+		editPost( { title } );
 	};
 
-	const updateCourseDescription = ( value ) => {
+	const updateCourseDescription = ( description ) => {
 		setWizardData( {
 			...wizardData,
-			courseDescription: value,
+			courseDescription: description,
```

Comment:
You could call the parameter `courseDescription` and pass the key and the value here:

```suggestion
			courseDescription,
```

But it's just a small detail, definitely nothing essential. =)